alea/src/autocorr.cpp

```diff
@@ -104,6 +104,16 @@ void autocorr_acc<T>::finalize_to(autocorr_result<T> &result)
 template class autocorr_acc<double>;
 template class autocorr_acc<std::complex<double> >;
 
+template <typename T>
+bool autocorr_result<T>::operator==(const autocorr_result &other) const
+{
+    if(nlevel() != other.nlevel()) return false;
+    for(size_t i = 0; i < nlevel(); ++i) {
+        if(level(i) != other.level(i))
```

Comment:
@krivenko  @mwallerb :  To address https://github.com/ALPSCore/ALPSCore/pull/446#issuecomment-365810182 and as a workaround for #462, could we compare `autocorr_result` equal even if contains `NaN` (or, at least, if it contains `NaN` at uppermost level)? This test failure is annoying. :)

Reply:
Let's just disable the test for the time being.